tests/platform_tests/daemon/test_pcied.py

```diff
@@ -52,6 +52,18 @@ def setup(duthosts, rand_one_dut_hostname):
         pytest.skip("{} is not enabled in {} {}".format(daemon_name, duthost.facts['platform'], duthost.os_version))
 
 
+@pytest.fixture(scope="module", autouse=True)
+def get_pcie_devices_tbl_key(duthosts, rand_one_dut_hostname, setup):
```

Comment:
@vvolam `setup` argument doesn't seem to be used?

Reply:
By including the setup() fixture as an argument, it is executed before the get_pcie_devices_tbl_key() fixture. This adjustment ensures that the setup() fixture runs first, confirming that the pcied daemon is active before proceeding.